src/cache/cache.ts

```diff
@@ -220,9 +220,9 @@ export function createCache(ctx: Context, bundleContext: IBundleContext): ICache
       const depPackage = packages[meta.packageId];
       // a required dependency is missing
       // verifying and external package of the current package
-      if (!meta) return false;
```

Comment:
As you can see a few lines above, `meta.packageId`. Which will throw if `meta` is falsey.  So this check here doesn't do anything.  The `return false` is unreachable.